src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class EmailValidatorTest {
+    private static final String EMAIL = "user@gmail.com";
+    private EmailValidator emailValidator;
+    private ConstraintValidatorContext constraintValidatorContext;
+
+    @BeforeEach
+    void setUp() {
+        emailValidator = new EmailValidator();
+        constraintValidatorContext = Mockito.mock(ConstraintValidatorContext.class);
+    }
+
+    @Test
+    void isValid_ok() {
```

Comment:
Add more test cases to check the validity of different email formats, such as missing '@', '.' or containing invalid characters. This will help ensure that the EmailValidator works correctly for various inputs. 🧪